pkg/container/qemu_runner.go

```diff
@@ -450,14 +449,17 @@ func createMicroVM(ctx context.Context, cfg *Config) error {
 	retries := 6000
 	try := 0
 	for try <= retries {
+		try++
```

Comment:
```suggestion
		if err := ctx.Err(); err != nil {
				return fmt.Errorf("checking SSH server: %w", err)		
		}
		
		try++
```

To ensure ctrl+C works.

Reply:
Thanks :heart: